test/libyul/yulOptimizerTests/loadResolver/keccak_reuse_mstore.yul

```diff
@@ -0,0 +1,21 @@
+{
+    let x := calldataload(0)
+    let a := keccak256(0x200, x)
+    sstore(a, 2)
+    // will disable loading for now, might improve later
+    mstore(0, 1)
+    let b := keccak256(0x20, x)
```

Comment:
Is this intentionally ``0x20`` instead of ``0x200``.

Reply:
Probbaly not, will fix.